### ISSUE TITLE
[Tests] In order to quickly prototype and tests functions we need fully functional test suite with a local test RPC running

### ISSUE THREAD

**Issue by pelsasser** [MARKETProtocol/MARKET.js#28]
### Why Is this Needed?

*Summary*: Rapid prototyping, testing, and interactions with web3

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Solution

*Summary*: Following typechains example tests

**Comment by eswarasai** [comment]
@pelsasser -- Probably we can create a new issue to increase the coverage percentage once #33 is in place and close this in favour of existing working tests?

**Comment by pelsasser** [comment]
yup! closing
